src/autolabel/transforms/base.py

```diff
@@ -71,7 +81,14 @@ async def apply(self, row: Dict[str, Any]) -> Dict[str, Any]:
                 # Cache hit
                 return output
 
-        output = await self._apply(row)
+        try:
+            output = await self._apply(row)
+            output[self.transform_status_column] = True
+        except Exception as e:
+            logger.error(f"Error applying transform {self.name()}. Exception: {e}")
+            output = {k: None for k in self.output_columns.values() if k is not None}
```

Comment:
Do we want to put this value as something standard like the NULL_TRANSFORM_TOKEN. Some string value instead of none might make sense